scapy/layers/tls/cert.py

```diff
@@ -128,32 +131,33 @@ def __call__(cls, obj_path, obj_max_size, pem_marker=None):
 
         if obj_path is None:
             raise Exception(error_msg)
+        obj_path = raw(obj_path)
 
         if (not b'\x00' in obj_path) and os.path.isfile(obj_path):
             _size = os.path.getsize(obj_path)
             if _size > obj_max_size:
                 raise Exception(error_msg)
             try:
                 f = open(obj_path)
-                raw = f.read()
+                raw_ = f.read()
```

Comment:
s/raw_/_raw/ do be consistent with the changes you made to the variables called bytes ?